src/biogeochem/CNBalanceCheckMod.F90

```diff
@@ -186,12 +186,22 @@ subroutine CBalanceCheck(this, bounds, num_soilc, filter_soilc, &
               (col_endcb(c) - col_begcb(c))
 
          ! check for significant errors
-         if (abs(col_errcb(c)) > 1e-7_r8) then
```

Comment:
These changes lower the error criterion when running matrixcn. We really don't want to do this without really good reasons to do it. So I'm testing it without these changes and we'll see if it can run without the lower tolerance. My hope is that this was a change that was needed during development, but is no longer necessary. If it is still necessary, we'll want to investigate it and make sure it's not much different than the current settings without matrixcn.

Reply:
I will try to fix error in the CN balance.

Reply:
This has been resolved.